examples/somersaultecu.py

```diff
@@ -523,6 +523,9 @@ class SomersaultSID(IntEnum):
             odx_id=OdxLinkId("somersault.multiplexer.flip_preference", doc_frags),
             short_name="flip_preference",
             long_name="Flip Preference",
+            description=None,
+            is_visible_raw=None,
+            sdgs=[],
```

Comment:
before you asked: The `DopBase` constructor had a default for the `sdgs` parameter, which lead to quite a few objects not being amendable using SDGs. Once more, I decided to do this as a "drive-by" fix.

The other two parameters are now necessary because this patches makes `DopBase` a dataclass which in turn became necessary because deriving a dataclass from a non-dataclass (`Multiplexer` was guilty of that attrocity) messes up the constructor arguments.